docs/events-api.asciidoc

```diff
@@ -9,8 +13,9 @@ Events can be:
 * Spans
 * Errors
 * Metrics
+* Metadata
```

Comment:
Metadata need to be sent as first line of every request, but they are not an event category. The meta information is the same for every event of one request and stored in ES as part of each of these events. 
E.g. information about the monitored `service` or the installed `agent` is stored with every event but for efficiency reasons only sent once per request as `metadata`. 
In summary - please do not list it as event here.

Reply:
Thanks, will remove